pkg/graveler/ref/commit_ordered_iterator.go

```diff
@@ -105,7 +105,7 @@ func (i *OrderedCommitIterator) Close() {
 	}
 }
 
-// getAllFirstParents returns a set of all commits that are not a first parent of other commit in the given repository.
+// getAllFirstParents returns a set of all commits following the first parent for a given repository.
```

Comment:
```suggestion
// getAllFirstParents returns a set of all commits that are the first parent of some other commit for a given repository.
```
It looks like it builds a list of commits that are the first parent of some other commit.